### ISSUE TITLE
Please Add Support for Cryptonight_R 

### ISSUE THREAD

**Issue by Romeo1984** [MoneroOcean/xmr-node-proxy#94]
Please Add Support for Cryptonight_R. My Node-Proxy stopped working today. Thanks.

**Comment by Blitss** [comment]
It was added. But it doesn't work, in my modified environment it produces invalid shares.

**Comment by djfinch** [comment]
I'm not digging monero for a while but made testing env now and looks OK to me...
So probably something on your side. Wrong config, I guess...
![image](https://user-images.githubusercontent.com/13107713/54083958-f59e7200-432a-11e9-8999-348c59eef619.png)

**Comment by Blitss** [comment]
Yeah, I figured. Because I'm using my own fork, I had to add block height to Miner job response. Therefore, you should update config & dependencies
